UKHomeOffice/nmsw-ui
# NMSW-324 Voyage Tasklist routes and skeleton page

# Ticket

NMSW-324

----

## AC

prototype: /mvp-2/draft

Page | URL | Link type | Link name | Link route | Next page
-- | -- | -- | -- | -- | --
Task list (Draft) | /report-voyage | text link | General Declaration (FAL 1) | /report-voyage/upload-general-declaration | Upload general declaration
Task list (Draft) | /report-voyage | text link | Crew details including supernumeraries (FAL 5) | /report-voyage/upload-crew-details | Upload crew details
Task list (Draft) | /report-voyage | text link | Any passenger details (FAL 6) | /report-voyage/passenger-details | Upload passenger details
Task list (Draft) | /report-voyage | text link | Supporting documents | /report-voyage/upload-supporting-documents | Upload supporting documents
Task list (Draft) | /report-voyage | text link | Check answers and submit | /report-voyage/check-your-answers | Check your answers
Task list (Draft) | /report-voyage | destructive button | Delete draft | /report-voyage/confirm-delete-draft | Are you sure delete draft



----

## To test

- run tests all should pass
- go to `/report-voyage`
- > page layout should match prototype
- click on General Declaration link
- > you should be taken to `/report-voyage/upload-general-declaration`
- go back to task list
- click on Crew
- > you should be taken to `/report-voyage/upload-crew-details`
- go back to task list
- click on Passenger
- > you should be taken to `/report-voyage/passenger-details`
- go back to task list
- click on Supporting docs
- > you should be taken to `/report-voyage/upload-supporting-documents`

----

## Notes
- all statuses and dynamic content are hardcoded
- the delete button does not take you to the right page as it does not exist yet (coming in next PR)
- the check your answers does not take you to the right page as it does not exist yet (coming in next PRs)
